exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/MetricAdapter.java

```diff
@@ -224,5 +263,30 @@ private static int estimateNumSamples(int numPoints, MetricDataType type) {
     return Collections.emptyList();
   }
 
+  private static Sample createSample(
+      String name,
+      List<String> labelNames,
+      List<String> labelValues,
+      double value,
+      Optional<Exemplar> exemplar) {
+    return exemplar
+        .map(e -> new Sample(name, labelNames, labelValues, value, toPrometheusExemplar(e)))
+        .orElseGet(() -> new Sample(name, labelNames, labelValues, value));
+  }
+
+  private static io.prometheus.client.exemplars.Exemplar toPrometheusExemplar(Exemplar exemplar) {
+    // TODO: Keep filtered attributes for prometheus?
+    if (exemplar.getSpanId() != null && exemplar.getTraceId() != null) {
+      return new io.prometheus.client.exemplars.Exemplar(
+          valueOf(exemplar),
+          exemplar.getRecordTimeNanos(),
+          "traceid",
```

Comment:
trade_id and span_id are what client_java uses.

Reply:
Is there a standard somewhere specifying this I can point to?  If not I'll just link against the Java source.

P.S. Thanks for the review!!!

Reply:
Not that I know of, the Java client is the first one and that's what it's using. The OM spec uses trace_id in an example.